src/nuclio/projects/project/functions/version/version-configuration/tabs/version-configuration-build/version-configuration-build.component.js

```diff
@@ -52,9 +54,14 @@
             ctrl.build.commands = lodash.get(ctrl.version, 'spec.build.commands', []);
             ctrl.build.commands = ctrl.build.commands.join('\n');
 
-            if (!lodash.isNil(lodash.get(ctrl.version, 'spec.build.dependencies'))) {
-                ctrl.build.dependencies = lodash.get(ctrl.version, 'spec.build.dependencies', []);
-                ctrl.build.dependencies = ctrl.build.dependencies.join('\n');
+            if (!lodash.isEmpty(lodash.get(ctrl.version, 'spec.build.runtimeAttributes.dependencies'))) {
+                ctrl.build.runtimeAttributes.dependencies = lodash.get(ctrl.version, 'spec.build.runtimeAttributes.dependencies', []);
+                ctrl.build.runtimeAttributes.dependencies = ctrl.build.runtimeAttributes.dependencies.join('\n');
+            }
```

Comment:
suggestion, instead of this entire `if`:
```js
ctrl.build.runtimeAttributes.dependencies = lodash.get(ctrl.version, 'spec.build.runtimeAttributes.dependencies', []).join('\n');
```

This way, `ctrl.build.runtimeAttributes.dependencies` will be equal to:
1. `[]` in case `ctrl.version.spec.build.runtimeAttributes.dependencies` is `undefined`
2. `[]` in case `ctrl.version.spec.build.runtimeAttributes.dependencies` is `[]`
3. `x\ny\nz` in case `ctrl.version.spec.build.runtimeAttributes.dependencies` is `['x', 'y', 'z']`